### ISSUE TITLE
Oikeusvaikutteisuus kaavakohteessa tai kaavamääräyksenä?

### ISSUE THREAD

**Issue by SannaJokela** [sykefi/kaavatietomalli#108]
Jos olemme tulkinneet oikein, vain kaavan aluerajauksella on tieto oikeusvaikutteisuudesta.

Miten tällaisessa tapauksessa tiedot kirjataan, jos yleiskaava on oikeusvaikutteinen, muuten paitsi tiettyjä osia koskien. Onko aluerajaus tällöin vain tuo oikeusvaikutteinen alue ja oikeusvaikutukseton alue on tavallaan toinen kaavan aluerajaus?

![image001](https://user-images.githubusercontent.com/23258285/144380541-08d2aec2-c8a0-4654-859f-469d9f6cac50.jpg)

Samoin tässä helsingin yleiskaavassa annettu yleismääräys viittaisi siihen, että kaava-alueesta vain osa on oikeusvaikutteinen ja osa ei. 

![image002 (1)](https://user-images.githubusercontent.com/23258285/144380611-8a6bd336-d7d6-403e-b6c9-938217745c88.jpg)

**Comment by ilkkarinne** [comment]
Joo, mielenkiintoinen tapaus. Yksinkertaisinta lienee, jos kunkin kaavan aluerajaus on se alue, jossa sen yleismääräykset ovat voimassa ja jonka sisällä sen kaavakohteet kokonaan sijaitsevat. Eli aluerajauksen geometriassa voi olla myös reikiä.

Tuo Hgin yleiskaavan määräys on vielä ehdollinen toisten osayleiskaavojen voimaantuloajankohtien suhteen, eli aika vänkä. Oisko siinä ajateltu sellaista takaporttia, että jos noita osayleiskaavoja ei saada läpi aikataulussa, niin tämä yleiskaava kuitenkin korvaisi näilläkin alueilla voimassaolevat vanhat yleiskaavat? Tätä mekanismia on jo aika vaikea mallintaa rakenteisesti
